src/map/atcommand.c

```diff
@@ -4100,6 +4100,8 @@ ACMD(mapinfo)
 		strcat(atcmd_output, msg_fd(fd, 1292)); // PrivateAirshipStartable |
 	if (map->list[m_id].flag.pairship_endable)
 		strcat(atcmd_output, msg_fd(fd, 1293)); // PrivateAirshipEndable |
+	if (map->list[m_id].flag.nopet)
```

Comment:
Please add ` != 0` or ` == 1`, since you're not checking a boolean value.

Reply:
i am on and off for this since the rest of the code are using the same style, perhaps leave it to future PR that fix all in one go ?
beside, aren't it look weird when there is only 1 line of code check it while the rest didn't?

Reply:
I agree with you that this needs to be changed for the other lines, too. But just because it wasn't done corretly before, you don't have to imitate it. Please fix that. :wink:

Reply:
fixed